src/errors/test-run/templates.js

```diff
@@ -61,12 +61,18 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
-    [TYPE.clientFunctionExecutionInterruptionError]: err => markup(err, `
+    [TYPE.clientFunctionInterruptedByPageUnloadError]: err => markup(err, `
         ${err.instantiationCallsiteName} execution was interrupted by page unload. This problem may appear if you trigger page navigation from ${err.instantiationCallsiteName} code.
 
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.clientFunctionInterruptedByDialogError]: err => markup(err, `
+        ${err.instantiationCallsiteName} execution was interrupted by a native ${err.dialogType} dialog with text "${err.dialogText}". Make sure that ${err.instantiationCallsiteName} code does not trigger native dialogs. If this dialog appears as a result of the previous test action, call a "handle..." function that handles this type of dialog after the action.
+
```

Comment:
Well, I didn't want to introduce a new term (like a *dialog handler* or *dialog handling function*) or enumerate all functions. So, this way I referred to dialog handling functions in general.
That's something we do in our main docs from time to time. Feels awkward?